konan_sdk/endpoints/base_endpoint.py

```diff
@@ -134,20 +165,37 @@ def headers(self) -> Dict:
 
 
 class KonanBaseDeploymentEndpoint(KonanBaseAuthenticatedEndpoint[ReqT, ResT]):
+    """Base Endpoint class for Konan endpoints that deal with deployments.
+
+    :param ReqT: type of request_object to use in .get() and .post() methods
+    :type ReqT: type
+    :param ResT: type of object that .get() and .post() methods return
+    :type ResT: type
+    """
     def __init__(
         self, api_url: str,
         deployment_uuid: str = None, user: KonanUser = None, **kwargs
     ) -> None:
+        """Initializes a Konan endpoint that deals with a deployment
+
+        :param api_url: base URL of Konan API
+        :type api_url: str
+        :param deployment_uuid: deployment uuid to use, defaults to None
+        :type deployment_uuid: str
+        :param user: user to authenticate as, defaults to None
+        :type user: KonanUser
+        :raises ValueError: raises ValueError when deployment_uuid is not valid
```

Comment:
might not be needed but eh might as well, should also indicate that it will be raised with invalid user too

Reply:
Fixed